### PR TITLE
Updating CLI verbiage to link to the Command Line Tools page

### LINKED ISSUE #40: Link to the new Command Line Tools page from About page instead of duplicating info
The new Command Line Tools page looks great!  Nice work, @jhadvig!

screen shot 2016-05-26 at 11 39 59 am

I noticed that we're duplicating some information on the About page about the CLI, however.

screen shot 2016-05-26 at 11 41 26 am

Perhaps we revise that last sentence to:

> With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal. To get started using the CLI, visit the [Command Line Tools](/command-line) page.

Thoughts, @jwforres, @spadgett, @benjaminapetersen, @sg00dwin?

### PR DESCRIPTION
**Pull request by rhamilto**
so as to avoid duplication on information.

Resolves #40

### CODE REVIEW

**Comment by rhamilto** [comment]
@jwforres or @spadgett, PTAL.

File: app/views/about.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h2 id="version">Version</h2>
 
                       <p>The <a target="_blank" href="{{'welcome' | helpLink}}">documentation</a> contains information and guides to help you learn about OpenShift and start exploring its features. From getting started with creating your first application, to trying out more advanced build and deployment techniques, it provides what you need to set up and manage your OpenShift environment as an application developer.</p>
 
-                      <p>With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal. To get started using the CLI you will need to <a target="_blank" href="command-line">download it and login</a>. For more information about the command line tools, check the <a target="_blank" href="{{'cli' | helpLink}}">CLI Reference</a> and <a target="_blank" href="{{'basic_cli_operations' | helpLink}}">Basic CLI Operations</a>.
+                      <p>With the OpenShift command line interface (CLI), you can create applications and manage OpenShift projects from a terminal. To get started using the CLI, visit <a href="/command-line">Command Line Tools</a>.
```
Comment (jwforres) [inline app/views/about.html]: drop the leading slash on the link, and run grunt build

**Comment by rhamilto** [comment]
@jwforres, fixed.

**Comment by jwforres** [comment]
[merge]

**Pull request merged**